services/app-web/src/pages/StateSubmission/RateDetails/RateDetailsSchema.ts

```diff
@@ -10,171 +10,176 @@ import {
 Yup.addMethod(Yup.date, 'validateDateFormat', validateDateFormat)
 
 const SingleRateCertSchema = (_activeFeatureFlags: FeatureFlagSettings) =>
-    Yup.object().when('.ratePreviouslySubmitted', {
-        // this first when is skipping all validations for a previously submitted rate.
-        is: 'YES',
-        then: Yup.object().shape({
-            linkRateSelect: Yup.string().defined('You must select a rate certification'),
```

Comment:
We changed `linkRateSelect` to `linkedRateDropdown` there are a couple places we still need to make the same changes.

RateDetailsV2.tsx

```ts
export type FormikRateForm = {
    ...
    linkRateSelect?: string
}
```

rateDetailsHelpers.ts

```ts
const convertGQLRateToRateForm = (getKey: S3ClientT['getKey'], rate?: Rate, parentContractID?: string): FormikRateForm => {
    const handleAsLinkedRate = rate && rate.parentContractID !== parentContractID // TODO: Make this a more sophisticated check for child-rates
    const rateRev = handleAsLinkedRate ? rate?.revisions[0] : rate?.draftRevision
    const rateForm = rateRev?.formData
    return {
        ...
        linkRateSelect: handleAsLinkedRate && rate?.id ? 'true' : undefined
    }
}
```